portfolio/src/main/webapp/json.js

```diff
@@ -0,0 +1,32 @@
+async function showExampleJSON()
+{
+  const responseFromServer = await fetch('/json');
+
+  const jsonData = await responseFromServer.json();
+
+  const JsonListElement = document.getElementById('json-container');
```

Comment:
Similar to Java, have any local variable names start with a lowercase letter ([style guide](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names)).
```suggestion
  const jsonListElement = document.getElementById('json-container');
```